src/index.ts

```diff
@@ -37,24 +37,25 @@ class ServerlessOfflineSSM {
   constructor(serverless: Serverless) {
     this.serverless = serverless
 
-    // only run offline plugin when offline command is passed in
-    if (this.serverless.processedInput.commands.indexOf('offline') < 0) {
+    if (!this.shouldRunPlugin()) {
+      console.log('The plugin "serverless-offline-ssm" only runs when offline.')
```

Comment:
I've added a message to indicate that the plugin is not running. I think it would be useful but I can remove it